code/modules/planar_lighting/light_effect_cast.dm

```diff
@@ -12,11 +12,15 @@
 
 	// Clear overlays, blank slate.
 	overlays.Cut()
```

Comment:
Assigning null would be even faster I believe. Since it's a BYOND voodoo list it'll get reset to blank when you give it an incorrect value.